metadata-ingestion/archived/source_docs/ldap.md

```diff
@@ -34,6 +34,26 @@ source:
     # Options
     base_dn: "dc=example,dc=org"
 
+    # Optional attribute mapping to allow ldap config differences across orgs
+    attrs_mapping:
+      urn: sAMAccountName
+
+      # user related attrs
+      fullName: cn
+      lastName: sn
+      firstName: givenName
+      displayName: displayName
+      manager: manager
+      mail: mail
+      departmentNumber: departmentNumber
+      title: title
+
+      # group related attrs
+      group_urn: cn
+      admins: owner
+      members: uniqueMember
+      displayName: name
```

Comment:
Won't this overwrite the display name attribute defined above? 

This makes me think we do need 2 distinct mappings:

userMappings
groupMappings

I should have called this out earlier - my apologies for that. What do you think?